packages/connectivity/README.md

```diff
@@ -33,15 +33,19 @@ We try to provide fixes, features and answers as soon as possible.
 If you would like to contribute to the SAP Cloud SDK, please make yourself familiar with our [contributing guidelines](https://github.com/SAP/cloud-sdk-js/blob/main/CONTRIBUTING.md) and follow the given instructions.
 
 ## Links
+- [Official support channel](https://github.com/SAP/cloud-sdk-js/issues)
```

Comment:
[pp] perhaps use this link because it will bring the user to the creation of issues: https://github.com/SAP/cloud-sdk-js/issues/new/choose